futures-util/src/future/pending.rs

```diff
@@ -33,6 +33,7 @@ impl<T> FusedFuture for Pending<T> {
 /// unreachable!();
 /// # });
 /// ```
+#[doc(alias = "never")]
```

Comment:
```suggestion
#[cfg_attr(docsrs, doc(alias = "never"))]
```

doc(alias) is unstable on our MSRV.